### DISCUSSION TITLE
Natifier rebuild?

### DISCUSSION THREAD

**Discussion by adhillon192** [kelyvin/Google-Messages-For-Desktop#75] [Q&A]
so github actions doesn't have an inbuilt trigger for workflows on specific time frames, will we require an external service for this?

**Comment by ghost** [comment]
Do you know if this project has been abandoned?  I just built and new Win 11 Pro PC and went to run this and got a message about it being old and shouldn't used used.

![image](https://github.com/user-attachments/assets/3320ea00-bfc3-427e-a1ae-f787002a34b8)

**Reply by hornetster** [reply]
Yep, I got the same now (May, 2025...), in Linux.
Moving on...

**Reply by adhillon192** [reply]
> Do you know if this project has been abandoned? I just built and new Win 11 Pro PC and went to run this and got a message about it being old and shouldn't used used.
> 
> ![image](https://private-user-images.githubusercontent.com/29755664/358954957-3320ea00-bfc3-427e-a1ae-f787002a34b8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTE4NDY0OTYsIm5iZiI6MTc1MTg0NjE5NiwicGF0aCI6Ii8yOTc1NTY2NC8zNTg5NTQ5NTctMzMyMGVhMDAtYmZjMy00MjdlLWExYWUtZjc4NzAwMmEzNGI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzA2VDIzNTYzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwNWZhMWMzNzkwYjNiZGZkODJiNTE1OTI2YjQ0NWI2ZjFiNDA5NTkyZThhYjI1MjMzYzAwN2EzM2ZiOWVkOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dbyLFWAfnR7lXNV4Il-i_XPMlliliIKvZSQpL8Iyci8)

yup looks abandoned now
